product_video/views/media_video_provider_view.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+<record id="media_video_provider_view_tree" model="ir.ui.view">
+    <field name="model">media.video.provider</field>
+    <field name="arch" type="xml">
+        <tree string="Video Provider">
+            <field name="name" />
+        </tree>
+    </field>
+</record>
+
+<record id="media_video_provider_view_form" model="ir.ui.view">
+    <field name="model">media.video.provider</field>
+    <field name="arch" type="xml">
+        <form string="Video Provider">
+            <field name="name" />
+        </form>
+    </field>
+</record>
+
+<record id="media_video_provider_view_search" model="ir.ui.view">
+    <field name="model">media.video.provider</field>
+    <field name="arch" type="xml">
+        <search string="Video Provider">
+            <field name="name" />
+        </search>
+    </field>
+</record>
+
+<record model="ir.actions.act_window" id="media_video_provider_action">
+    <field name="name">Video Provider</field>
+    <field name="type">ir.actions.act_window</field>
+    <field name="res_model">media.video.provider</field>
+    <field name="view_mode">tree,form</field>
+    <field name="search_view_id" ref="media_video_provider_view_search" />
+    <field name="domain">[]</field>
+    <field name="context">{}</field>
+</record>
+
+<menuitem
+        id="media_video_provider_menu"
+        parent="sale.prod_config_main"
+        sequence="20"
+        action="media_video_provider_action"
+    />
```

Comment:
if a more generic approach is used, we could imagine an extra new menu item, named video, with all the videos, and the providers.
so, no need to depends on sale module.